source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -509,6 +633,183 @@ def test_stop_area_without_modes(self):
+            'limit': 10,
+            'pt_dataset': 'main_routing_test',
+            'depth': 0
+        }
+
+        url += "?{}".format(urlencode(params, doseq=True))
+        mock_requests = MockRequests({
+            url: (BRAGI_MOCK_BOBETTE_DEPTH_ZERO, 200)
+        })
+        with mock.patch('requests.get', mock_requests.get):
+            response = self.query_region("places?q=bobette&pt_dataset=main_routing_test&type[]=stop_area"
+                                         "&type[]=address&type[]=poi&type[]=administrative_region&depth=0")
+
+            r = response.get('places')
+            assert len(r) == 1
+            assert r[0]['name'] == "bobette's label"
+            assert r[0]['embedded_type'] == "poi"
+            poi = r[0]['poi']
+            assert poi['label'] == "bobette's label"
+            assert poi['properties']["amenity"] == "bicycle_rental"
+            assert poi['properties']["capacity"] == "20"
+            assert poi['properties']["ref"] == "12"
+            #Empty administrative_regions not displayed as in kraken
+            assert not poi.get('administrative_regions')
+            #Address absent as in kraken
+            assert not poi.get('address')
+
+    def test_autocomplete_call_with_depth_one(self):
+        url = 'https://host_of_bragi/autocomplete'
+        params = {
+            'q': u'bobette',
+            'type[]': [u'public_transport:stop_area', u'street', u'house', u'poi', u'city'],
+            'limit': 10,
+            'pt_dataset': 'main_routing_test',
+            'depth': 1
+        }
+
+        url += "?{}".format(urlencode(params, doseq=True))
+        mock_requests = MockRequests({
+            url: (BRAGI_MOCK_BOBETTE_DEPTH_ONE, 200)
+        })
+        with mock.patch('requests.get', mock_requests.get):
+            response = self.query_region("places?q=bobette&pt_dataset=main_routing_test&type[]=stop_area"
+                                         "&type[]=address&type[]=poi&type[]=administrative_region&depth=1")
+
+            r = response.get('places')
+            assert len(r) == 1
+            assert r[0]['name'] == "bobette's label"
+            assert r[0]['embedded_type'] == "poi"
+            poi = r[0]['poi']
+            assert poi['label'] == "bobette's label"
+            assert poi['properties']["amenity"] == "bicycle_rental"
+            assert poi['properties']["capacity"] == "20"
+            assert poi['properties']["ref"] == "12"
+            poi_admins = poi['administrative_regions']
+            assert len(poi_admins) == 1
+            assert poi_admins[0]['id'] == "admin:fr:02000"
+            assert poi_admins[0]['insee'] == "02000"
+            assert poi_admins[0]['label'] == "Bobtown (02000)"
+            assert poi_admins[0]['coord']['lat'] == "48.8396154"
+            assert poi_admins[0]['coord']['lon'] == "2.3957517"
+
+            address = poi['address']
+            assert address['coord']['lat'] == '8.98312e-05'
```

Comment:
Please have different values in lat/lon so we know if we mixed them (same everywhere)